deploy/scripts/src/authentication/test.ts

```diff
@@ -346,13 +365,13 @@ export function signUp(): void {
   sleep(1)
 
   // B01_SignUp_11_ContinueAccountCreated
-  timeGroup(groups[12], () => {
+  timeGroup(groups[15], () => {
```

Comment:
**RP Stub Route**

- Using RP Stub, the call from Auth is redirected to Orchestration (OIDC) and then to the RP stub. We need to split this in 3 parts so that response time can be measured separately for these 3 redirections

**Using Orch Stub**

- Using Orch stub, the call is redirected from Auth to Orchestrator stub. 

The current redirection handling in the script will work for Orch stub scenario but not for RP stub. We would need to add a condition that if the route is RP then the 3 redirects are captured.

Reply:
Condition added as per the comment